RELEASE_PROCEDURE.md

```diff
@@ -103,6 +103,7 @@ Other tasks:
  * Manually update the files mentionning the current release version (```README.md``` and ```RELEASE_NOTES.md```)
  * Update the branch name for the Travis status logo
  * Change the `packageDatadirScmVersion` parameter in the root `pom.xml` to `20.1`
+ * Replace `99.master` by `${build.closest.tag.name}` in the root `pom.xml` so that debian packages have the right version
```

Comment:
IIUC, this line should be removed after first execution.

Reply:
i think it stays forever in the stable branches, we're talking about https://github.com/georchestra/georchestra/pull/3471/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R633 right ?

from my understanding only master should 'hardcode' `99.master` and the other branches should have `${build.closest.tag.name}`